### PR TITLE
@l2succes => Upper desktop footer [WIP]

### PR DESCRIPTION
**Pull request by mzikherman**
screen shot 2017-03-01 at 5 00 53 pm

Getting there, I skipped a couple of the footer links that are supposed to open modals.

Still very much a WIP, as I add other sections here I may reorganize.

### CODE REVIEW

**Comment by yuki24** [comment]
We should probably generalize and componentize the list part so we can re-use it anywhere.

**Comment by mzikherman** [comment]
Yea, I wasn't sure how generic/deep to component-ize things here. Idk, is it worth it do you think? This is just a `div` of `a` tags... I'm not sure what else to do with it (or if it's actually needed/reusable).

**Comment by alloy** [comment]
Let’s not extract code until we actually need to use it elsewhere, at which time we’ll better know how to generalise it.

File: src/components/layout/desktop_footer.tsx [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+import * as fonts from "../../assets/fonts"
+
+const asideWidth = 28
+const columnWidth = (100 - asideWidth) / 4
+
+const Footer = styled.footer`
+  ${fonts.secondary.style}
+  background-color: white;
+  padding: 20px 0;
+  .column, .column-aside {
```
Comment (alloy) [inline src/components/layout/desktop_footer.tsx]: Does this prefix the class names in any way / is this recommended styled-components usage? I thought the idea was to style the individual elements.

Comment (mzikherman) [inline src/components/layout/desktop_footer.tsx]: I did inspect the DOM and it seemed alright to me.

The generated CSS is prefixed by the overall component one, so this did 👍 seem to work and not pollute too much.

screen shot 2017-03-01 at 5 59 33 pm

**Comment by mzikherman** [comment]
> Let’s not extract code until we actually need to use it elsewhere, at which time we’ll better know how to generalise it.

👍 to that

File: src/components/layout/desktop_footer.tsx (hunk 1 above)
Comment (mzikherman) [inline src/components/layout/desktop_footer.tsx]: Oh, or did you mean have them each like:

```
<Footer>
  <Column>
  ...
  </Column>
</Footer>
```

where `Column` is a styled div?

Comment (alloy) [inline src/components/layout/desktop_footer.tsx]: Good point 👌

Comment (alloy) [inline src/components/layout/desktop_footer.tsx]: Yup, that’s what I meant.

Comment (l2succes) [inline src/components/layout/desktop_footer.tsx]: I agree with @alloy. I'd say create a `Column` component with its styling encapsulated

**Comment by mzikherman** [comment]
Whoops, I guess this is premature!

**Pull request closed without merging**